internal/builders/golang/build.go

```diff
@@ -119,6 +119,12 @@ func (*Builder) Build(ctx *context.Context, build config.Build, options api.Opti
 	}
 	cmd = append(cmd, gcflags...)
 
+	tags, err := processFlag(ctx, artifact, env, strings.Join(build.Tags, " "))
```

Comment:
According to https://golang.org/cmd/go/#hdr-Compile_packages_and_dependencies:

```
-tags tag,list
	a comma-separated list of build tags to consider satisfied during the
	build. For more information about build tags, see the description of
	build constraints in the documentation for the go/build package.
	(Earlier versions of Go used a space-separated list, and that form
	is deprecated but still recognized.)
```

I would prefer comma-separated tags.

```suggestion
	tags, err := processFlag(ctx, artifact, env, strings.Join(build.Tags, ","))
```